subcommittees/mentoring/PracticumProposal.md

```diff
@@ -0,0 +1,192 @@
+connections which may lead to more discussions between instructors at a later stage. Also, by having
+people already within the organization mentoring future instructors, they get to know people already
+inside of the organization. 
+
+Fourth, as a bonus we get a lot of people working through the material, which can only be a benefit to
+the lesson development process. It might draw more people into helping out with lesson development, and
+could lower the bar for people developing new lessons.
+
+With these goals in mind, we have developed a post-instructor training practicum.  
+
+## Practicum elements
+
+The instructor training practicum aims to facilitate the transition from the introductory instructor
+training  to actually teaching workshops, by doing the following activities: 
+
+### Mentoring
+
+Future instructors, or trainees, form small groups and are assigned a mentor.  Over a period of a 3-4
+weeks, they work through the core material* for Software and Data Carpentry on their own, as if they
+were workshop participants.  Each week, the group meets to discuss their experiences with the lesson
+material and the mentor receives feedback and provides guidance.  The primary goal is not that the
+trainees should be prepared enough to teach the module, but to become familiar with it and gain skills.  
+
+### Mini-lessons
+
+Each trainee chooses a topic relevant to Software and Data Carpentry (can be one of our lessons or
+their own provided their mentor approves) and teaches a 1 hour lesson at their local institution to an
+audience of their choice.  This lesson should be at least partially observed by a mentor and feedback
+provided.  Whenever possible, the mentor observing the mini-lesson should be the mentor of the previous
+mentoring element.  
+
+### General comments
+
+These two elements need not happen consecutively, but in the first implementation of this practicum
+they will be consecutive, not parallel (see timeline).  
+
+These two elements will be required to become a fully badged Software or Data Carpentry Instructor.  
+
+## Implementation Logistics
+
+### Mentoring Groups
+
+For the first practicum, all recent participants in instructor training (online 8 week, December 2 day)
+will be invited to join.  This program may be extended to include all newly trained instructors after
+we have figured out how it works.
+
+Candidates will sign up via a form that requests the following information: 
+
+- Wants to teach Software Carpentry and/or Data Carpentry 
+- Whether they want to do Python or R  
+- group member preference from their training (if any) 
+- time zone + geographic location (nearest airport), preferred time of day for meetings (daytime/evening) 
+- Language preferences (if any). 
+- When they went through instructor training 
+ 
+Based on this data, the mentoring committee will form groups of 3-4 people, prioritizing in the order
+listed above.  
+
+### Mentors
```

Comment:
> They will certainly get the title senior instructor, and they will also be offered a bit of training regarding being mentors too. I had not thought of offering them priority of the workshops that they would like to go to, but it is a pretty easy perk to give them. Maybe we could also offer to give them certificates or something along those lines? Those could for instance be used to show teaching ability when applying for jobs... (I think. Not too familiar with the US academic system, but I assume it would help?)

Could we offer once a year priority to this instructors when host can cover airplane tickets
and there isn't a strong group nearby?
For example,
we get contacted by someone from Spain and we don't need anyone available there
so someone will need to flight to Spain. If the host didn't make special requests
(e.g. experience with X) we will send one of the instructors that mentored others.